addons/sale_timesheet/tests/test_so_line_determined_in_timesheet.py

```diff
@@ -129,6 +129,7 @@ def test_sol_determined_when_project_is_employee_rate(self):
         timesheet = self.env['account.analytic.line'].create({
             'name': 'Test Line',
             'unit_amount': 1,
+            'auto_account_id': self.analytic_account_sale.id,
             'employee_id': self.employee_manager.id,
             'project_id': self.project_employee_rate.id,
             'task_id': task.id,
```

Comment:
the timesheet line was created without account_id previously? was it working or there was a default value somehow?

Reply:
It was working because the constraint was an SQL one and it never got flushed.
If you tried to flush, it would fail so the test was wrong.

Reply:
lol